db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160323170519) do
+ActiveRecord::Schema.define(version: 20160324124129) do
```

Comment:
It appears that you didn't actually add the unique index, or you generated this before you ran the mirgation.